finance-service-api/src/main/kotlin/com/ajaxproject/financeservice/service/FinanceService.kt

```diff
@@ -0,0 +1,32 @@
+package com.ajaxproject.financeservice.service
+
+import com.ajaxproject.financeservice.repository.FinanceRepositoryImpl
+import com.ajaxproject.financeservice.enums.Finance
+import com.ajaxproject.financeservice.enums.Finance.EXPENSE
+import com.ajaxproject.financeservice.enums.Finance.INCOME
+import com.ajaxproject.financeservice.models.MongoFinance
+import org.bson.types.ObjectId
+import org.springframework.stereotype.Service
+
+@Service
+class FinanceService(
+    private val financeRepositoryImpl: FinanceRepositoryImpl,
```

Comment:
isn't injecting FinanceRepository interface enough? do we need implementation here?